src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -1395,6 +1395,24 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <remarks>https://redis.io/commands/zrevrange</remarks>
         RedisValue[] SortedSetRangeByRank(RedisKey key, long start = 0, long stop = -1, Order order = Order.Ascending, CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Takes the specified range of elements in the sorted set of the source key and stores
+        /// them in a new sorted set at the destination key.
```

Comment:
```suggestion
        /// Takes the specified range of elements in the sorted set of the <paramref name="sourceKey"/> and stores
        /// them in a new sorted set at the <paramref name="destinationKey"/>.
```
Also: let's reverse the order of these args please - IMO source -> destination is a much more natural way to read a C# API, despite the args in Redis being in reverse of this.